Note non-standard behavior of view transitions for forms set to post (#5828)
withastro/docs: src/content/docs/en/guides/view-transitions.mdx

By default, your page will transition after a form is submitted, even if you are using `event.preventDefault()` to prevent this behavior.

> Please note that if your method is set to `post`, the default behavior of web browsers is to submit your form data encoded as as `application/x-www-form-urlencoded`, but the default behavior of Astro is to submit your form data as `multipart/form-data`. If you want to match the default behavior of web browsers make sure to set the `enctype` attribute accordingly:

Reviewer: ```suggestion
By default, Astro submits your form data as `multipart/form-data` when your `method` is set to `POST`. If you want to match the default behavior of web browsers, use the `enctype` attribute to submit your data encoded as `application/x-www-form-urlencoded`:
```

```astro title="src/components/Form.astro"

> <form action="/contact" method="post" enctype="application/x-www-form-urlencoded">

Reviewer: ```suggestion
<form action="/contact" method="POST" enctype="application/x-www-form-urlencoded">
```

Author: It's a stylistic thing. The docs currently have both. I'll leave it to you to decide :)

<img width="426" alt="image" src="https://github.com/withastro/docs/assets/174864/4fd50764-0a97-428f-ae3d-c5d30f56fabd">
